### DISCUSSION TITLE
[Question][Hw0] Cannot connect to KCLOUD VM

### DISCUSSION THREAD

**Discussion by miraliahmadli** [prosyslab-classroom/cs348-information-security#544] [Q&A]
I cannot connect to my KCLOUD VM.  It asks for the password, but when I write the password, it says permission denied.  My kcloud id is CS448-54. Can you please fix this issue?

I am using port 22. Also, I cannot connect to ports 80 and 443. It says connection refused.

Can you please reset my password for vm instance? I cannot access.

**Comment by Junseon00** [comment]
Regarding the account of the KCLOUD, contact to kcloud@kaist.ac.kr
